api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/IntensiveMonitoringDataSet.java

```diff
@@ -99,6 +99,126 @@ public DataSetDefinition constructIntensiveMonitoringDataSet() {
                 "revisionEndDate=${revisionEndDate},location=${location}"),
             "revisionEndDate=${revisionEndDate},location=${location}"),
         "");
+    // 13.2 DEN
```

Comment:
where did you add your dimension?

Reply:
There is no need for dimensions on these specific indicators as they use age calculated inside the methods instead of dimensions from the dataset